src/component/AgendaItem.jsx

```diff
@@ -0,0 +1,223 @@
+import React, { Component } from "react";
+import PropTypes from "prop-types";
+import { connect } from "react-redux";
+import { Link } from "react-router-dom";
+import agenda_item_received from "../actions/Form";
+import { bindActionCreators } from "redux";
+
+//I can just get the id from the param and use to to fetch from Application state's agenda agendaitems
+//Now I just need to design a page and put informations on them
+import { Button, Card, Container } from "semantic-ui-react";
+
+class AgendaItem extends Component {
+  constructor() {
+    super();
+
+    this.recommendationReducer = this.recommendationReducer.bind(this);
+    this.summaryReducer = this.summaryReducer.bind(this);
+    this.showForm = this.showForm.bind(this);
+    this.goToForm = this.goToForm.bind(this);
+  }
+
+  componentDidMount = () => {};
+
+  timeConverter(UNIX_timestamp) {
```

Comment:
This is a general helper function so can you extract this into a different file and put it into a folder called say `util/helper.js`